docs/specification.md

```diff
@@ -688,32 +689,48 @@ data VoteParam = VoteParam
   , voteAmount :: Natural
   }
 
+data Permit = Permit
+  { key :: PublicKey
+  , signature :: Signature
+  }
+
 data Parameter proposalMetadata
-  = Vote [VoteParam]
+  = Vote [(VoteParam, Maybe Permit)]
 ```
 
 Parameter (in Michelson):
 
 ```
 (list
-  (pair %vote
-    (nat %proposal_key)
-    (pair
-      (bool %vote_type)
-      (nat %vote_amount)
+  (pair :permit_protected
+    (pair %vote
```

Comment:
This `%vote` annotation looks quite weird to me, shouldn't it apply to the topmost `list`?

Reply:
Oh, fair